content/en/docs/Installation/_index.md

```diff
@@ -413,14 +413,21 @@ helm upgrade kyverno --namespace kyverno kyverno/kyverno --version <version_numb
 
 To uninstall Kyverno, use either the raw YAML manifest or Helm. The Kyverno deployment and all CRDs will be removed, including any reports.
 
-### Uninstall Kyverno with YAML manifest
+### Option 1 - Uninstall Kyverno with YAML manifest
 
 ```sh
 kubectl delete -f https://raw.githubusercontent.com/kyverno/kyverno/main/definitions/release/install.yaml
 ```
 
-### Uninstall Kyverno with Helm
+### Option 2 - Uninstall Kyverno with Helm
 
 ```sh
 helm uninstall kyverno --namespace kyverno kyverno/kyverno
 ```
+
+### Clean up Webhook Configurations
```

Comment:
Can you put a brief sentence or two here explaining to users why and/or when they might need to do a manual clean-up of the webhooks? Otherwise, it's a bit confusing.

Reply:
@chipzoller - I added the explanation, kindly review and let me know if there is any change.